content/master/concepts/managed-resources.md

```diff
@@ -635,21 +635,145 @@ my-rds-instance      True    True     my-custom-name       11m
 
 ### Creation annotations
 
-Providers create new managed resources with the
-`crossplane.io/external-create-pending` annotation.
+In some rare situations a provider can forget that it created a resource. When
+this happens the provider can't manage the resource.
 
-The Provider applies the `crossplane.io/external-create-succeeded` or
-`crossplane.io/external-create-failed` annotation after making the external API
-call and receiving a response. 
+{{<hint "tip">}}
+Crossplane calls resources that a provider creates but doesn't manage _leaked
+resources_.
+{{</hint>}}
 
-{{<hint "note" >}}
-If a Provider restarts before creating the `succeed` or `fail` annotations the
-Provider can't reconcile the managed resource. 
+Providers set three creation annotations to avoid and detect leaked resources:
 
-Read Crossplane [issue #3037](https://github.com/crossplane/crossplane/issues/3037#issuecomment-1110142427)
-for more details 
-{{< /hint >}}
+* {{<hover label="creation" line="8">}}crossplane.io/external-create-pending{{</hover>}} -
+  The last time the provider was about to create the resource.
+* {{<hover label="creation" line="9">}}crossplane.io/external-create-succeeded{{</hover>}} -
+  The last time the provider successfully created the resource.
+* `crossplane.io/external-create-failed` - The last time the provider failed to
+  create the resource.
+
+
+Use `kubectl get` to view the annotations on a managed resource. For example, an
+AWS VPC resource:
+
+```yaml {label="creation" copy-lines="2-9"}
+$ kubectl get vpc my-vpc
+apiVersion: ec2.aws.upbound.io/v1beta1
+kind: VPC
+metadata:
+  name: my-vpc
+  annotations:
+    crossplane.io/external-name: vpc-1234567890abcdef0
+    crossplane.io/external-create-pending: "2023-12-18T21:48:06Z"
+    crossplane.io/external-create-succeeded: "2023-12-18T21:48:40Z"
+```
+
+A provider uses the
+{{<hover label="creation" line="7">}}crossplane.io/external-name{{</hover>}}
+annotation to find a resource in an external system, for example AWS.
+
+If the provider can't find a managed resource in an external system, it thinks
+the resource doesn't exist. When the provider thinks a resource doesn't exist
+it creates the resource.
+
+Some external systems don't let a provider specify a resource's name when the
+provider creates it. Instead the external system generates an unpredictable name
```

Comment:
```suggestion
provider creates it. Instead the external system generates a name
```

It's only unpredictable from the POV of the provider. This could trip someone up if they say "hey, the external system docs say this is how they generate a name, right here, so I guess this doesn't apply"

Reply:
I don't think that's true. In many cases it's unpredictable to any outside observer.

I wanted to say non-deterministic. Happy to say that instead if we feel comfortable with that terminology.

Reply:
Updated to say nondeterministic.

This detail feels important to me, as an explanation of why we can't do better here. If there was a way to predict the name, e.g. because it would be generated in a predictable way, we wouldn't need to do this potentially frustrating annotation dance.